storage/kvstore_test.go

```diff
@@ -831,3 +815,58 @@ func (i *fakeIndex) Compact(rev int64) map[revision]struct{} {
 	return <-i.indexCompactRespc
 }
 func (i *fakeIndex) Equal(b index) bool { return false }
+
+func TestTxnPut(t *testing.T) {
+	sliceSize := 100
+	bytesSize := 30
+	keys := createMultiBytesSlice(bytesSize, sliceSize)
+	vals := createMultiBytesSlice(bytesSize, sliceSize)
+
+	s := newStore(tmpPath)
+	defer cleanup(s, tmpPath)
+
+	for i := 0; i < size; i++ {
```

Comment:
I would prefer n to size. N indicates times, which is more suitable here in my opinion.